test/storage-luatest/storage_1_1_test.lua

```diff
@@ -263,3 +263,47 @@ test_group.test_on_bucket_event = function(g)
         box.space.data2:drop()
     end)
 end
+
+--
+-- Test that services for all background fibers are created
+-- and work properly (gh-107).
+--
+test_group.test_basic_storage_service_info = function(g)
+    local uuid = g.replica_1_a:exec(function()
+        -- Test that all services save states
+        local info = ivshard.storage.info({with_services = true})
+        ilt.assert_not_equals(info.services, nil)
+        ilt.assert_not_equals(info.services.gc, nil)
+        ilt.assert_not_equals(info.services.recovery, nil)
+        ilt.assert_not_equals(info.services.rebalancer, nil)
+        -- Routes applier service is created as soon as it's needed
+        ilt.assert_equals(info.services.routes_applier, nil)
+
+        local internal = ivshard.storage.internal
+        internal.errinj.ERRINJ_APPLY_ROUTES_STOP_DELAY = true
+        return box.info.cluster.uuid
+    end)
+
+    g.replica_2_a:exec(function(uuid)
+        -- Send bucket to create disbalance in
+        -- order to test routes applier service
+        local bid = _G.get_first_bucket()
+        ivshard.storage.bucket_send(bid, uuid)
```

Comment:
Nit: I would advice to do this (to catch the error and to protect the test from being flaky):
```Lua
local ok, err = ivshard.storage.bucket_send(bid, uuid)
ilt.assert_equals(err, nil)
ilt.assert(ok)
```
(I usually intentionally test `err == nil` first, because if we would fail on `ok`, then the error wouldn't be reported in luatest output.)

Reply:
Fixed